app/controllers/users_controller.rb

```diff
@@ -1,9 +1,14 @@
+require 'set'
+
 class UsersController < ApplicationController
   include SocialAccountsHelper
-  before_action :authorize
+  include UsersHelper
+  before_action except: %i[show index search]
   helper_method :generate_link, :supported_social_networks
 
-  def show; end
+  def show
```

Comment:
This change should not be necessary if you would leave `before_action :authorize` this would not be necessary

Reply:
I changed it back